doc/how_to_guides/pick_ik/pick_ik_tutorial.rst

```diff
@@ -0,0 +1,130 @@
+pick_ik Kinematics Solver
+=========================
+
+``pick_ik`` is an inverse kinematics (IK) solver compatible with MoveIt 2,
+developed by PickNik Robotics. It is designed to provide robust and customizable IK solutions,
+offering a wide range of features.
+
+The solver in ``pick_ik`` is a reimplementation of ``bio_ik``, integrating a global optimizer and a local optimizer.
+The global optimizer utilizes evolutionary algorithms to explore alternative solutions within the solution space
+and identify global optima. Building upon the results obtained by the global optimizer, the local optimizer applies
+gradient descent for iterative refinement of the solution. It takes a potential optimum solution provided by the global
+optimizer as input and aims to improve its accuracy and ultimately convergence to the most optimum solution.
+
+Getting Started
+---------------
+Before proceeding, please ensure that you have completed the steps outlined in the :doc:`Getting Started Guide </doc/tutorials/getting_started/getting_started>`.
+
+Additionally, it is required to have a MoveIt configuration package specifically tailored to your robot.
```

Comment:
Here's an example repo with MoveIt configurations that we like to use: https://github.com/ros-planning/moveit_resources/tree/ros2/panda_moveit_config

Following Sebastian's suggestion, you can use this example config, and create your own `kinematics.yaml` file for pick_ik.